DOC: Add further aspects of dMRI to discussion

jhlegarreta opened the pull request:
Add further aspects of dMRI to discussion.

kaitj commented on _extras/discuss.md:
```diff
@@ -1,6 +1,40 @@
+The lesson presented a way to recover the fODFs on single shell data using the
+Constrained Spherical Deconvolution method. This is also called Single-Shell
+Single-Tissue (SSST) CSD since only the white matter tissue is used in the
+process. Having multiple shell data allows to decompose the contribution of
```
Would slightly rephrase to "allows for decomposition of the contribution of different tissues"

jhlegarreta replied:
Done in cae0647.

kaitj commented on _extras/discuss.md:
```diff
@@ -1,6 +1,40 @@
+different tissues (as different macroscopic tissues have unique b-value
+dependencies) to the fODF using the so-called Multi-Shell Multi-Tissue CSD
+(MSMT-CSD) method. Performing a multi-tissue decomposition using single
+shell data (Single-Shell 3-Tissue CSD, SS3T-CSD) has also been proposed.
```
Wondering if references here would be helpful for MSMT and SS3T?

jhlegarreta replied:
Done in cae0647.

kaitj commented on _extras/discuss.md:
```diff
@@ -1,6 +1,40 @@
+to study the *in vivo* human brain. Although the resolution and the information
+made available from the dMRI data is improving over time (using more powerful
+MRI machines and other new acquisition sequences), it still does not allow to
+reach the level of detail that invasive method offer.
```
invasive methods

jhlegarreta replied:
Done in cae0647.

kaitj reviewed (commented):
Looks good, just a couple of changes to phrasing / spelling.

Also think that maybe some references could be helpful. Could just say they are outside the scope of these lessons, but here are references if interested?

jhlegarreta commented:
> Also think that maybe some references could be helpful. Could just say they are outside the scope of these lessons, but here are references if interested?

Addressed in cae0647. I have not mentioned that they are out of the scope of the lesson; IMHO the fact that we added the discussion highlights the relevance. I think the provided information is enough so that learners are aware of it.

kaitj reviewed (approved):
> I have not mentioned that they are out of the scope of the lesson; IMHO the fact that we added the discussion highlights the relevance. I think the provided information is enough so that learners are aware of it.

That is fair - I do agree that it is relevant, and important especially if someone going through this is planning on applying tractography for their research. By "out of scope", I meant they should probably read the paper for details as outside of making them aware these methods exist, we probably wouldn't going go into detail about this (at this time anyway). Probably not the correct choice of phrasing from my last message. 

Nonetheless, sounds & looks good to me! Happy to merge once @josephmje has a chance to look / comment on the PR.

jhlegarreta commented:
:+1: There references were added, so I think that everything is in place.

josephmje commented on _extras/discuss.md:
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
+structural MRI parcellations). This is useful, among others, to study diseases
+(neurological or not) affecting the brain or different aspects related to its
+functional patterns (such as attention, learning, etc.). Similarly, studying the
+structural wiring of the brain allows to study its evolution and do comparative
```
```suggestion
structural wiring of the brain allows one to study its evolution and do comparative
```

josephmje commented on _extras/discuss.md:
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
+structural MRI parcellations). This is useful, among others, to study diseases
+(neurological or not) affecting the brain or different aspects related to its
+functional patterns (such as attention, learning, etc.). Similarly, studying the
+structural wiring of the brain allows to study its evolution and do comparative
```
```suggestion
structural wiring of the brain allows us to study its evolution and do comparative
```

jhlegarreta replied:
Done in fd0e235.

josephmje commented on _extras/discuss.md:
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
+The power of diffusion MRI lies in its ability to offer a non-invasive method
+to study the *in vivo* human brain. Although the resolution and the information
+made available from the dMRI data is improving over time (using more powerful
+MRI machines and other new acquisition sequences), it still does not allow to
```
```suggestion
MRI machines and other new acquisition sequences), it still does not allow us to
```

jhlegarreta replied:
Done in fd0e235.

josephmje commented on _extras/discuss.md:
```diff
@@ -24,6 +24,41 @@ The mentioned concepts are common across different 3D visualization packages.
+to study the *in vivo* human brain. Although the resolution and the information
+made available from the dMRI data is improving over time (using more powerful
+MRI machines and other new acquisition sequences), it still does not allow to
+reach the level of detail that invasive methods offer.
```
```suggestion
reach the level of detail that invasive methods, such as chemical tracers or anatomical sectioning, offer.
```

jhlegarreta replied:
Done in fd0e235.
